types/enginko/MCF-LW12CO2E/meta.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "Enginko",
+  "version": "1.0.0",
+  "manufacturer": "MCF-LW12CO2E",
+  "url": "https://enginko.com/en/solutions/mcf-lw12co2/",
+  "description": "This device reads temperature, humidity, pressure, light intensity, air quality (expressed as IAQ or bVOC value) and CO2 and sends collected data over the LoRaWAN® network. 3 traffic light LEDs for air quality display with adjustable brightness. Ideally suited for a wide range of applications such as building automation, condition monitoring, predictive maintenance, security, healthcare and many more use cases.",
```

Comment:
@J2thatsme please use the description defined in the task:  This device reads temperature, humidity, pressure, light intensity, air quality (expressed as IAQ or bVOC value) and CO2. 3 traffic light LEDs for air quality display with adjustable brightness. Ideally suited for a wide range of applications such as building automation, condition monitoring, predictive maintenance, security, healthcare and many more use cases.